src/meson.d

```diff
@@ -0,0 +1,3 @@
+version (meson) {
```

Comment:
Adding `src/meson.d` would allow the statement `import meson;`. Can this be removed?